src/diagram.rs

```diff
@@ -409,6 +409,12 @@ pub struct Diagram {
 
     #[serde(flatten)]
     pub extensions: Option<ExtensionSettings>,
+
+    /// Optional text to describe the workflow.
+    pub description: Option<String>,
+
+    /// Examples of inputs that can be used with this workflow.
+    pub example_inputs: Option<Vec<String>>,
```

Comment:
A few concerns here:

* For each example input, we should also allow a description, so users can have some context about the input.
* The data type of the input should not be `String`, it should be `JsonMessage`. The `String` type is too ambiguous, i.e. should the value be interpreted as a literal character string or as the text of a JSON message?
* There's no need to wrap `Option` around `Vec`. We can tag this field with `#[serde(default, skip_serializing_if = "is_default")]`. That way the field can just be skipped for empty vectors.

Overall we should see something like

```rust
pub struct ExampleInput {
    pub value: JsonMessage,
    pub description: String,
}
```

```rust
#[serde(default, skip_serializing_if = "is_default")]
example_inputs: Vec<ExampleInput>,
```